src/components/comments/viewComment.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/commentStyles/newComment.scss';
+
+let styleSubmit =styles.submit;
```

Comment:
why do we need this @huxaiphaer ?

Reply:
I declared this style to be accessible to other elements, code climate was complaining about its duplication.